### ISSUE TITLE
Undefined Feature - label name="feature"

### ISSUE THREAD

**Issue by jalona** [allure-framework/allure1#858]
Allure: 1.4.24.RC3

feature

```
# language: ru

@master
Функционал: Пример на примере

    Сценарий: Сценарий один
      Пусть мы находимся на главной странице
```

xml

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1474620334864" stop="1474620335311" version="1.4.24.RC3" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Сценарий один</name>
    <title>Сценарий один</title>
    <test-cases>
        <test-case start="1474620334877" stop="1474620334879" status="passed">
            <name>Пусть мы находимся на главной странице</name>
            <title>Пусть мы находимся на главной странице</title>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="PC077469"/>
                <label name="thread" value="7372@PC077469.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="story" value="Сценарий один"/>
        <label name="feature" value="Undefined Feature"/>
        <label name="framework" value="CucumberJVM"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
```

**Comment by just-boris** [comment]
If there something wrong with XML file content, the issue should belong to CucumberJVM adapter:
https://github.com/allure-framework/allure-cucumber-jvm-adaptor

I'll keep this open, but it would be better it you re-create this ticket in allure-framework/allure-cucumber-jvm-adaptor repo

**Comment by clicman** [comment]
Fixed in allure-cucumber-jvm-adaptor 1.6.1
